### ISSUE TITLE
How to get output video?

### ISSUE THREAD

**Issue by TranDangTin** [ValentinH/react-easy-crop#360]
I seeing the example to crop video as the link below
https://codesandbox.io/s/react-easy-crop-for-videos-lfhme

However, I don't know How to get output video?
Any ideal?

Thanks.

**Comment by ValentinH** [comment]
I don't think this is possible to do it in the browser. You need to do this server side with something like Ffmpeg.
